src/components/AuditLogsTable/AuditLogsTable.tsx

```diff
@@ -122,20 +153,43 @@ const AuditLogsTable = ({ showModel = false }: Props) => {
 
   return (
     <div className="audit-logs__table">
-      <Button
-        onClick={() => {
-          fetchAuditEvents();
-          setQueryParams({ page: "1" });
-        }}
-      >
-        Refresh
-      </Button>
+      <div className="audit-logs__filter">
+        <Button
+          onClick={() => {
+            setQueryParams({ page: "1" });
+            fetchAuditEvents(limit);
+          }}
+        >
+          Refresh
+        </Button>
+        <div className="audit-logs__pagination">
+          <Pagination
+            onForward={() => {
+              setQueryParams({ page: `${Number(page) + 1}` });
+            }}
+            onBack={() => {
+              setQueryParams({ page: `${Number(page) - 1}` });
+            }}
+            forwardDisabled={tableData.length <= limit}
```

Comment:
It could also be worth adding a comment here about the fact that we fetch an extra item so that we can check if there are more pages.